### PR TITLE
Ngrok

### PR DESCRIPTION
**Pull request by joaomilho**

### CODE REVIEW

**Comment by joaomilho** [comment]
@ilonashub @AmirKremer do you know Case's github as well?

**Comment by AmirKremer** [comment]
https://github.com/ctaintor @ctaintor

**Comment by joaomilho** [comment]
@ctaintor

File: lib/shoot/ngrok.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+require 'securerandom'
+
+module Shoot
+  class Ngrok
+    extend Forwardable
+    def_delegators :@process, :start, :stop, :exited?
+
+    def initialize(port = 3000)
+      @process = ChildProcess.build("ngrok", "-log=stdout", "-subdomain=#{subdomain}", port.to_s)
+      start
```
Comment (AmirKremer) [inline lib/shoot/ngrok.rb:12]: Do we need to close it somehow?

File: README.md [hunk 2]
```diff
@@ -145,6 +145,38 @@ You got the idea.
...
+
+1. Install it from [https://ngrok.com/download](https://ngrok.com/download)
+
+2. Enable subdomains by registering.
+
+3. Use the `Shoot::Ngrok` class in your test, like this:
+
+``` ruby
+  def my_test
+    my_server = Shoot::Ngrok(12345)
```
Comment (AmirKremer) [inline README.md:160]: Would it be possible to run multiple Ngrok servers? (We sometimes need another one for PGW on a different port)

File: lib/shoot/ngrok.rb (hunk 1 above)
Comment (joaomilho) [inline lib/shoot/ngrok.rb:12]: No. The idea of the ChildProcess is exactly that when your process dies, it dies together.

File: README.md (hunk 2 above)
Comment (joaomilho) [inline README.md:160]: Yes, I currently need 2. But doing it manually sux a lot.

**Comment by AmirKremer** [comment]
Can't wait to use it! :+1:

**Comment by joaomilho** [comment]
Cool, I'll merge it ASAP. BTW, I'm considering moving this to klarna/shoot. WDYT?

**Pull request merged**